src/converters/convert.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Given an input is a converter, it will return a converted result.
+ *
+ * @param {...function} converters - This is the converters.
+ *
+ * @returns {object} - The converted result.
+ */
+export default function convert(...converters) {
+    if (!converters.length) {
+        throw new Error('You need to use at least one converter.');
+    }
+
+    return (input, defaultValue, name) => {
+        const result = converters.map((converter) => converter(input, defaultValue, name));
```

Comment:
Can modify this so we only use a converter if the previous one failed, basically not precompute the result array.